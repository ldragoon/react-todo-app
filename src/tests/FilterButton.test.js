import React from 'react';
import renderer from 'react-test-renderer';
import FilterButton from '../components/FilterButton';

test('Dashed border appears when button is pressed', () => {
  const component = renderer.create(
    <FilterButton
      key="test-key"
      name="test-name"
      isPressed="false"
      setFilter="Completed"
    ></FilterButton>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});