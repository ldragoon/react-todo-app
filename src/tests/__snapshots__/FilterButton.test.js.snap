// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashed border appears when button is pressed 1`] = `
<button
  aria-pressed="false"
  className="btn toggle-btn"
  onClick={[Function]}
  type="button"
>
  <span
    className="visually-hidden"
  >
    Show 
  </span>
  <span>
    test-name
     
  </span>
  <span
    className="visually-hidden"
  >
     tasks
  </span>
</button>
`;
